multicluster/controllers/multicluster/member/serviceexport_controller.go

```diff
@@ -281,11 +291,16 @@ func (r *ServiceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 			eps.Subsets = []corev1.EndpointSubset{svcIPAsSubset}
 		}
 	} else {
-		newSubsets := common.FilterEndpointSubsets(eps.Subsets)
+		var newSubsets []corev1.EndpointSubset
+		if r.endpointSliceEnabled {
+			newSubsets = eps.Subsets
+		} else {
+			newSubsets = filterEndpointSubsets(eps.Subsets)
+		}
 		if epsInstalled {
 			installedEp := epsObj.(*epInfo)
 			if apiequality.Semantic.DeepEqual(newSubsets, installedEp.subsets) {
-				klog.InfoS("Endpoints has been converted into ResourceExport and no change, skip it", "endpoints",
+				klog.InfoS("Service's Endpoints (PodIP or ClusterIP) has been converted into ResourceExport and no change, skip it", "Service",
```

Comment:
This branch is only for `EndpointIPTypePodIP`? Why we talk about PodIP too?

Reply:
No, both EndpointIPTypePodIP and EndpointIPTypeClusterIP will go through this branch, because the `newSubsets` is actually all ready endpoints from either Endpoints/EndpointSlices or ClusterIP of Service before this. 
No matter it's EndpointIPTypePodIP and EndpointIPTypeClusterIP, we will create an Endpoint type of ResourceExport.

Reply:
Really? This is the "else" branch of `if r.endpointIPType == common.EndpointIPTypeClusterIP`?

Reply:
The installed EP is actually new defined structure, the purpose is to track the change of available addresses. 
```
epInfo := &epInfo{
		name:      eps.Name,
		namespace: eps.Namespace,
		subsets:   eps.Subsets,
	}
```
I probably should rename the installedEp to installedAddresses? this `eps` is not directly means the  Endpoints object but a new formed Endpoints based on the type of `EndpointIPTypePodIP`  and `EndpointIPTypeClusterIP`.
For EndpointIPTypePodIP, it's all ready addresses from Endpoints object, for EndpointIPTypeClusterIP, it's ClusterIP from Service object.

Reply:
Could you answer my question:
> This is the "else" branch of `if r.endpointIPType == common.EndpointIPTypeClusterIP`?

If yes, it should be for ClusterIP Endpoints only? Could you check the code to see any mistake here?

Reply:
oh, yes, you are right, I suppose to move this out of the else branch. Good catch, thanks!